tests/lowering/mod.rs

```diff
@@ -343,7 +343,7 @@ fn gat_higher_ranked_bound() {
     lowering_success! {
         program {
             trait Fn<T> {}
-            trait Ref<'a, T> {}
+            struct Ref<'a, T> {}
```

Comment:
Why change this?

Reply:
It was wrong to begin with. Traits are not types (and we now draw a sharper distinction). You can make a `dyn` type to reference `Ref`, but that is not what this was supposed to be testing.